src/toil/statsAndLogging.py

```diff
@@ -231,8 +233,34 @@ def callback(fileHandle: Union[IO[bytes], IO[str]]) -> None:
                 )
                 cls.writeLogFiles(jobNames, messages, config=config)
 
+            try:
+                jobs = stats.jobs
+            except AttributeError:
```

Comment:
Could this eat genuine misconfiguration errors?  Could there be a comment here about what the intended reason for it passing here?